src/patternfly/demos/Compass/examples/Compass.md

```
@@ -191,6 +202,11 @@ wrapperTag: div
 ```
 
 ### Multiple sections
+
+This demo places multiple sections within the main Compass section, with each section containing a card view. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)